docs/src/main/sphinx/connector/phoenix.md

```diff
@@ -47,11 +47,22 @@ The following Phoenix-specific configuration properties are available:
 
 | Property name                      | Required | Description                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
Ideally we change this whole list into a list-table .. but that could be a separate commit before or after .. or even a separate PR